packages/commerce-events-collector-segments/src/index.ts

```diff
@@ -34,4 +41,26 @@ const callProxyService = async (ecid: string) => {
     setAdobeCommerceSegmentCookies(segmentMembershipIds);
 };
 
+const initializeGetAEPSegements = async () => {
+    console.log("initializeGetAEPSegements");
+
+    // let retry = true;
+    const ALLOY_COMMAND = "sendEvent"; //"getIdentity"
+
+    if (window.hasOwnProperty("alloy")) {
+        console.log("Alloy is available.");
+
+        window.alloy(ALLOY_COMMAND).then((result: AlloyIdentity) => {
+            console.log("result: ", result);
+            const alloyDestinationsSegments = result?.destinations[0].segments;
+            const segmentList = alloyDestinationsSegments.map((segmentItem) => segmentItem.id);
+
+            console.log("alloyDestinationsSegments: ", alloyDestinationsSegments);
+            console.log("segmentList: ", segmentList);
```

Comment:
the console statements don't bother me exactly but are we running a linter on these changes and in this package? seems like something that should ahve warned and failed and required an `/*eslint-ignore*/`